modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java

```diff
@@ -203,9 +203,13 @@ public void postProcessContextBooleanFilter(
 		}
 
 		if (GetterUtil.getBoolean(attributes.get("secure"))) {
-			long commerceChannelId = GetterUtil.getLong(
+			long commerceChannelGroupId = GetterUtil.getLong(
```

Comment:
Just a reminder, we need to update the new indexer as well when it gets merged: https://github.com/liferay-commerce/liferay-portal/pull/4101